cypress/support/pages/yves/multi-factor-auth/repositories/b2b-mp-multi-factor-auth-repository.ts

```diff
@@ -0,0 +1,51 @@
+import { injectable } from 'inversify';
+import { MultiFactorAuthRepository } from '../multi-factor-auth-repository';
+
+@injectable()
+export class B2bMpMultiFactorAuthRepository implements MultiFactorAuthRepository {
+  private readonly SELECTORS = {
+    VERIFICATION_CODE_INPUT: 'input[name="codeValidationForm[authentication_code]"]',
+    VERIFY_BUTTON: 'form[name="codeValidationForm"] button[type="submit"]',
+    VERIFICATION_POPUP: '.js-multi-factor-authentication-handler__popup-content',
+    FORM_BOX: '.form.box',
+    ACTIVATE_FORM: 'form[name^="activateForm"]',
+    DEACTIVATE_FORM: 'form[name^="deactivateForm"]',
+    SUBMIT_BUTTON: 'button[type="submit"]',
+  };
+
+  getVerificationCodeInput(): Cypress.Chainable {
+    return cy.get(this.SELECTORS.VERIFICATION_CODE_INPUT);
+  }
+
+  getVerifyButton(): Cypress.Chainable {
+    return cy.get(this.SELECTORS.VERIFY_BUTTON);
+  }
+
+  getVerificationPopup(): Cypress.Chainable {
+    return cy.get(this.SELECTORS.VERIFICATION_POPUP, { timeout: 15000 });
```

Comment:
can we move this timeout into the page class?

Reply:
No, putting it inside `should` does not work. So far, I got rid of it to see how it works in CI, but if the tests fail then I'll have to revert that